api_methods.py

```diff
@@ -0,0 +1,132 @@
+"""Python auto.ria.com API.
```

Comment:
I would name the module `autoria_api`, or even just `autoria`. Current `api_methods` is too generic.
It is clearer if we have `from autoria import get_average` than `from api_methods import get_average`.